README.md

```diff
@@ -15,6 +15,13 @@ When using drush on the host you need:
 ```
 export IS_DDEV_PROJECT=true
 ```
+or this in your `settings.local.php`:
+
+```
+putenv("IS_DDEV_PROJECT=true");
+```
+
+Running `drush` commands in the containers is faster, however, dependingon your use, you may find it more convenient to run them on the host. [Read more about installing Global Drush and using this add-in](https://selwynpolit.github.io/d9book/drush#global-drush). Note. The host drush version doesn't matter much since it is only used to find the proper `drush` (most likely within /vendor/bin) and call it. This means you always run the version of `drush` that you installed in your project using composer.
```

Comment:
```suggestion
Running `drush` commands in the containers is faster, however, depending on your use, you may find it more convenient to run them on the host. [Read more about installing computer-side Drush and using this add-in](https://selwynpolit.github.io/d9book/drush#global-drush). Note. The host drush version doesn't matter much since it is only used to find the proper `drush` (most likely within /vendor/bin) and call it. This means you always run the version of `drush` that you installed in your project using composer. To use this technique you must have the same version of `php` and preferably the same version of `composer` on the host that you have inside the container.
```

Reply:
You could mention the use of brew-php-switcher to help switch php versions as needed